code/game/objects/effects/landmarks.dm

```diff
@@ -3,98 +3,199 @@
 	icon = 'icons/mob/screen_gen.dmi'
 	icon_state = "x2"
 	anchored = 1.0
+	layer = 4
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 
-/obj/effect/landmark/New()
-
-	..()
+/obj/effect/landmark/Initialize(mapload)
+	. = ..()
 	set_tag()
 	invisibility = 101
+	GLOB.landmarks_list += src
 
-	switch(name)			//some of these are probably obsolete
-		if("start")
-			GLOB.newplayer_start += loc
-			qdel(src)
+/obj/effect/landmark/newplayer_start //There should only be one of these, in the lobby art area
+	name = "start"
 
-		if("wizard")
-			GLOB.wizardstart += loc
-			qdel(src)
+/obj/effect/landmark/newplayer_start/New() //If this isn't new, you spawn in the corner of the map
```

Comment:
Consider using `INITIALIZE_IMMEDIATE`. 

```dm
INITIALIZE_IMMEDIATE(/obj/effect/statclick)

/obj/effect/statclick/Initialize(mapload, text, target)
	. = ..()
	name = text
	src.target = target
```

It equates to 
```dm
#define INITIALIZE_IMMEDIATE(X) ##X/New(loc, ...){\
    ..();\
    if(!initialized) {\
        args[1] = TRUE;\
        SSatoms.InitAtom(src, args);\
    }\
}
```

But it makes more sense and also wont trip the "annihilate /New()" regex.

Reply:
Tested and it works, comitting.